nimare/meta/cbma/mkda.py

```diff
@@ -166,12 +166,7 @@ def _compute_weights(self, ma_values):
         return weight_vec
 
     def _compute_summarystat_est(self, ma_values):
-        # Note: .dot should be faster, but causes multiprocessing to stall
-        # on some (Mac) architectures. If this is ever resolved, we can
-        # replace with the commented line.
-        # return ma_values.T.dot(self.weight_vec_).ravel()
-        weighted_ma_vals = ma_values * self.weight_vec_
-        return weighted_ma_vals.sum(0)
+        return ma_values.T.dot(self.weight_vec_).ravel()
```

Comment:
This seems like an edge case that is not worth the slow down, so I went w/ the faster version

Reply:
I can test it out tomorrow.

Reply:
`tests.test_meta_mkda.test_MKDADensity_approximate_null` went from 7.37s to 6.49s after making the change on my MacBook Pro (OSX 12.3.1). Maybe the architecture Tal was referring to is the M1 chip? Another possibility is that the problem was unique to multiprocessing, and when I switched to joblib it resolved the issue without anyone noticing.

In case anyone wants to dig further into it, the commit where the workaround was added is 68c515a5511f9baaa265bc24026a12defe86d4fb.